docs/tutorials/services/try-viam-color-detection.md

```diff
@@ -49,20 +63,26 @@ Select the **Config** tab, and click on **Services**.
 Scroll to the **Create Service** section.
 To create a [vision service](/services/vision/):
 
-1. Select `vision` as the **Type**.
-2. Enter `my_color_detector` as the **Name**.
-3. Select **Color Detector** as the **Model**.
-4. Click **Create Service**.
+1. Select `Vision` as the **Type**.
+1. Enter `my_color_detector` as the **Name**.
+1. Select **Color Detector** as the **Model**.
+1. Click **Create Service**.
+1. In the resulting vision service panel, click the color picker box to set the color to be detected.
+   For this tutorial, enter the following values:
 
-In your vision service's panel, fill in the **Attributes** field.
+   Color | Value
+   ----- | -----
+   R | 122
+   G | 79
+   B | 92
 
-``` json {class="line-numbers linkable-line-numbers"}
-{
-      "segment_size_px": 100,
-      "detect_color": "#7a4f5c",
-      "hue_tolerance_pct": 0.06
-}
-```
+   Or use the contextual menu in the lower-right of the pop-up window to switch to hex mode, and enter the hex code `#7a4f5c`.
```

Comment:
This is too specific since the UI varies depending on browser and OS. For me, hex code goes in the top :D 
```suggestion
   Or enter the equivalent hex code: `#7a4f5c`
```

Reply:
Ah sorry, you put this right in the ticket!! 🤦  Fixed, thank you!